### ISSUE TITLE
Misleading instructions for DNS setup

### ISSUE THREAD

**Issue by pmajkutewicz** [atech/postal#409]
OK, I'm trying to look around postal, so I have basic/minimal setup using docker.
I've added testdomain.com and I see:
![domains](https://user-images.githubusercontent.com/12598208/32996964-10556e04-cd8a-11e7-98d7-539ed415795a.png)
As I understand **postal.example.com** is taken from portal.yml. 

This suggests that only one domain is supported. In the matter of fact when I click "Check my records are correct" the testdomain.com DNS is checked and that is great. But still my first impression was that I can only use one domain - configured in postal.yml

TL;DR; I think that DNS example should contain setup domain name instead of posta.yml web.host

**Comment by willpower232** [comment]
I think you have misunderstood the setup process.

You are supposed to create DNS records which point to your server like this https://github.com/atech/postal/wiki/Domains-&-DNS-Configuration

Then you update postal.yml to include those records. i.e. spf.postal.example.com should be changed to a domain that you own and can point to your server (e.g. spf.postal.testdomain.com).

Presuming you are a typical user, you should only have one installation of Postal with many domains on it that each have an SPF record pointing to your installation of Postal.

**Comment by pmajkutewicz** [comment]
> I think you have misunderstood the setup process.

This is highly possible :).

Lets assume that I have two domains maindomain.com and devdomain.com.
After entering all my first domain details into postal.yml webpage will sugest me to add:
`v=spf1 a mx include:spf.postal.maindomain.com -all`

Than I'll have second domain setup, and webpage will suggest me to enter 
"`v=spf1 a mx include:spf.postal.maindomain.com -all`" not `v=spf1 a mx include:spf.postal.devdomain.com -all` 
Is that it the way supposed to be?

(Assuming that both domains will point to same postal host)

**Comment by willpower232** [comment]
That is indeed the way it is supposed to be.

There is nothing stopping you from having multiple subdomains for your Postal server but for simplicity, its easier to just remember one.

I'm not sure if Postal would stop you from sending emails if the DNS checks fail but they are there to help you guarantee your emails will be delivered.

**Comment by pmajkutewicz** [comment]
@willpower232 
Ah, ok, than I totally misunderstood those instructions.
Thanks You for clarification.
